.gitignore

```diff
@@ -1,26 +1,24 @@
-.idea
-*.iml
-.project
-.settings
-.vscode
+# ignore everything to avoid committing build artifacts
+**/*
```

Comment:
Yikes...  Maybe _slightly_ cleaner would be `**/*.js` and `**/*.js.map`?  I think that should cover all build artifacts.

Also maybe this is a terrible idea?

Reply:
Yeah, there are different ways to go about ignoring, this didn't work anyway. NPM reads `.gitignore` so it just includes the stuff we _don't_ want. :sweat_smile:

Reply:
`docs` is a build artifact too, so need to unignore that.